test/sagemaker_tests/pytorch/training/integration/sagemaker/test_distributed_operations.py

```diff
@@ -237,6 +243,7 @@ def test_sanity_efa(n_virginia_ecr_image, efa_instance_type, n_virginia_sagemake
     Tests pt mnist command via script mode
     """
     validate_or_skip_smmodelparallel(n_virginia_ecr_image)
+    validate_or_skip_smmodelparallel_efa(n_virginia_ecr_image)
```

Comment:
Why run the less restrictive validation at all when the more restrictive validation for `efa` filters out more cases?